simple/poll.c

```diff
@@ -216,7 +216,7 @@ int main(int argc, char **argv)
 	if (!hints)
 		return EXIT_FAILURE;
 
-	while ((op = getopt(argc, argv, "h" ADDR_OPTS CS_OPTS INFO_OPTS)) != -1) {
+	while ((op = getopt(argc, argv, "h" CS_OPTS INFO_OPTS)) != -1) {
```

Comment:
ADDR_OPTS is alreadyh included in CS_OPTS. This chagne is to just to avoid double ADDR_OPTS in getopt's options string